/pb_torque_controller:
  ros__parameters:
    Torque_constant: 0.438
    N_Spec: [0.0, 300.0, 600.0, 1000.0, 1700.0, 4400.0, 6790.0]
    Torque_Spec: [0.0, 0.0, 0.8, 2.9, 5.6, 9.8, 16.6]
